.github/workflows/release.yml

```diff
@@ -7,25 +7,43 @@ on:
 
 jobs:
     build:
-
         runs-on: ubuntu-latest
         services:
           mongodb:
             image: mongo:4.2.0
-            ports:
-              - 27017:27017
+            env:
+              MONGO_INITDB_ROOT_USERNAME: admin
+              MONGO_INITDB_ROOT_PASSWORD: admin
```

Comment:
I think these are default passwords, not harm to expose. If so shall we add a comment saying it

Reply:
These credentials are created by default when the container starts. there is no harm in keeping them public since this is used only using testing in a docker container